iceoryx_posh/test/moduletests/test_posh_runtime.cpp

```diff
@@ -22,6 +22,7 @@
 #include "iceoryx_posh/popo/subscriber.hpp"
 #include "iceoryx_posh/runtime/posh_runtime.hpp"
```

Comment:
This include could be removed

Reply:
Also please adapt the copyright header

Reply:
I would keep it since there are direct calls to `iox::runtime::PoshRuntime`.

Copyright is already correct :)